_posts/en/newsletters/2020-03-04-newsletter.md

```diff
@@ -0,0 +1,226 @@
+  which is very seldom changed by developers.
+
+    Discussion in the meeting covered the use of asserts and logging in Bitcoin
+    Core, performance and benchmarking, and how to test and review changes to
+    consensus-critical code.
+
+- **[Remove calls to Chain::Lock methods][review club 17954]:** This PR
+  ([#17954][Bitcoin Core #17954]) changes the way that the wallet component
+  calls functionality on the node component, and makes it more straightforward to
+  implement [a future change][Bitcoin Core #16426] that reduces the wallet's
+  ability to pause the node's processing.
+
+    Discussion in the meeting covered the differences between the wallet's view
+    of the block chain and the node's view of the block chain, and synchronization
+    issues between the wallet and node components.
+
+- **[Try to preserve outbound block-relay-only connections during restart][review club 17428]:**
+  This PR ([#17428][Bitcoin Core #17954]) adds _anchor connections_ to Bitcoin
+  Core, which the node will preferably try to reconnect to between restarts.
+  These persistent connections could mitigate some classes of [eclipse
+  attacks][topic eclipse attacks].
+
+    Discussion in the meeting covered remote restart attacks, DoS vectors and
+    privacy on the peer-to-peer network.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure.
+Please consider upgrading to new releases or helping to test release
+candidates.*
+
+- [LND 0.9.1][] is a new minor version release that doesn't contain any
+  new features but which fixes several bugs, including a bug that can
+  result in "erroneous force closes between nodes".
+
+- [Bitcoin Core 0.19.1][] (release candidate)
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #17985][] net: Remove forcerelay of rejected txs FIXME:moneyball
+
+- [Bitcoin Core #17264][] changes the default in RPCs that create or
+  process Partially Signed Bitcoin Transactions ([PSBTs][topic psbt])
+  to include any known [BIP32][] HD wallet derivation path.  This allows
+  any other wallets that later process the PSBT to use that information
+  to select the appropriate signing key or verify that a change output
+  pays the correct address.  Those wanting to keep their derivation
+  paths private may disable sharing using the `bip32_derivs`
+  parameter.
+
+- [C-Lightning #3490][] adds a `getsharedsecret` RPC that will derive a
+  shared secret from the combination of the local node's private key and
+  a user-specified public key.  This shared secret is derived the same
+  way other shared secrets are derived in LN (SHA256 digest of the
```

Comment:
s/LN/the LN/?

The style guide permits the acronym, but it may be more readable here to write "in the Lightning Network"